### PR TITLE
dev-java/jna: no longer fail with jdk-11

### PR DESCRIPTION
**Pull request by vaukai**
Package-Manager: Portage-3.0.18, Repoman-3.0.2
Signed-off-by: Volkmar W. Pogatzki <gentoo@pogatzki.net>

### CODE REVIEW

File: dev-java/jna/jna-4.2.2-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+# fine. Someone bisect it please. :)
+RESTRICT="test"
+
+CDEPEND="dev-libs/libffi"
+
+RDEPEND="${CDEPEND}
+	virtual/jre:1.8"
+
+DEPEND="${CDEPEND}
+	virtual/jdk:1.8
```
Comment (fordfrog) [inline dev-java/jna/jna-4.2.2-r2.ebuild:34]: the message says it does not fail with jdk-11 but the ebuild is restricted to `jdk:1.8`, i'm confused :-) could you please explain that?

Comment (vaukai) [reply dev-java/jna/jna-4.2.2-r2.ebuild:34]: > the message says it does not fail with jdk-11 but the ebuild is restricted to `jdk:1.8`, i'm confused :-) could you please explain that?

It was unlimited `>=virtual/jdk-1.7` before, causing it to fail with jdk-11.  And setting `-1.8:*` didn't work.
Will fix that soon ...

Comment (fordfrog) [reply dev-java/jna/jna-4.2.2-r2.ebuild:34]: ah, so it does not fail with jdk-11 because it does not compile with jdk-11 anymore, got that :-)

**Comment by fordfrog** [comment]
@vaukai thanks! also dropped old.

**Pull request closed without merging**